### PR TITLE
modify ISP pipeline for better image quality

### PR DESCRIPTION
**Pull request by sjbgg**
1. move the nr2d module before local contrast and sharpen(currently empty), so that final images can get better details
2. modified requirements.txt, removed specific version, tested fine

### CODE REVIEW

**Comment by xx-isp** [comment]
Thanks @sjbgg. Will test the quality of the new pipeline on a few more images before accepting the PR.

File: requirements.txt
Comment (xx-isp) [inline requirements.txt:1]: @sjbgg the requirements.txt file is generated automatically using the [gen_requirements.py](https://github.com/xx-isp/infinite-isp/blob/main/util/gen_requirements.py) file without any manual tweaking. It gives a good reference of the library versions on which the pipeline will certainly run. A user can use this requirements.txt to install the exact versions of libraries on which the pipeline will run using the [how-to-run](https://github.com/xx-isp/infinite-isp#how-to-run) guide.

Comment (sjbgg) [reply requirements.txt:1]: I see. I ran into some problems while installing so I deleted all the version numbers. And it automatically installed the latest version from the server.
I agree the version numbers are good for reference though.

**Comment by bakhtawar-10xe** [comment]
PR is outdated now after pipeline Version 1.1 Upgrade

**Pull request closed without merging**